tests/capi/unittest_capi_inference.cc

```diff
@@ -4754,7 +4744,7 @@ TEST (nnstreamer_capi_element, get_property_string_04_n)
 
   pipeline = g_strdup_printf (
       "appsrc name=appsrc ! "
-      "other/tensor,dimension=(string)1:1:1:1,type=(string)float32,framerate=(fraction)0/1 ! "
+      "other/tensor,dimension=(string)1,type=(string)float32,framerate=(fraction)0/1 ! "
       "tensor_filter name=filter_h framework=tensorflow-lite model=%s ! tensor_sink name=tensor_sink",
```

Comment:
For such testing, let's keep the previous test cases and add new cases with shortened dimension declarations.

Reply:
Let's talk about it tomorrow anytime when I'm available in the office.